cmd/registry-replacer/main.go

```diff
@@ -278,6 +304,15 @@ func replacer(
 			if err := pruneUnusedReplacements(config, allReplacementCandidates); err != nil {
 				return fmt.Errorf("failed to prune unused replacements: %w", err)
 			}
+
+			// to prune base images we'll need the fully-resolved config.
+			resolvedConfig, err := configResolver()
+
```

Comment:
Nit: The empty line between the call and the error handling bothers me, the two are usually directly after each other as they belong together :P

Reply:
Lol, I can remove that space. My OCD wanted it there, but I definitely see your point.